Source/Particles/MultiParticleContainer.H

```diff
@@ -322,7 +322,7 @@ protected:
             info.EnableTiling(pc_src.tile_size);
         }
 
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
         info.SetDynamic(true);
```

Comment:
Do we still `SetDynamic` manually in code or generally as an input option?

Reply:
What input option?

Reply:
I mean `warpx.do_dynamic_scheduling = 1`, is that it?

Reply:
Discussed on slack: those are set for particle loops generally since we know that those have computational inbalances.

We don't set this  generally atm for MFIter loops, but should independently of this PR review if the three locations where we set this for MFIter loops are still up-to-date or if we want to remove those (or add to more locations).